frontend/public/index.html

```diff
@@ -22,6 +22,8 @@
   <link rel="preload" href="https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap"
     rel="stylesheet">
 
+  <link rel="stylesheet" href="./index.css" />
```

Comment:
```suggestion
  <link rel="preload" href="./index.css" as="style" />
```

성능향상을 위해서 이렇게 쓰는 것도 가능하대용 👍

Reply:
오오.. 넘 좋아요

Reply:
사용할 때 preload는 미리 파일을 load하는거지, 적용하는 건 아니라서, 해당 구문을 변경하지 않고, 티케가 제안해주신 내용을 위에 추가했습니다~~